dist/lightstep-datasource/partials/query.editor.html

```diff
@@ -47,6 +47,19 @@
       </info-popover>
     </div>
 
+    <div class="gf-form">
+      <label class="gf-form-label">Resolution</label>
+      <input type="text"
+             class="gf-form-input width-6"
+             ng-model="ctrl.target.resolution"
+             ng-blur="ctrl.onChangeInternal()"
+             placeholder="">
+      </input>
+      <info-popover mode="right-absolute">
+        Length of time represented by each "point" in the timeseries (default is computed based on the size of the graph)
```

Comment:
would be nice to have the popover give information about how to format the resolution